File: Plugins/Aspose_Tasks_Java_for_Ruby/lib/asposetasksjava/Projects/verifyprojectstructure.rb
module Asposetasksjava
  module VerifyProjectStructure
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new(data_dir + 'test_tasks.mpp')

        if project.check()
            puts "Project is ok"
        else
            puts "Project is not ok"
        end
    end
  end
end

File: Plugins/Aspose_Tasks_Java_for_PHP/src/aspose/tasks/WorkingWithProjects/VerifyProjectStructure.php
<?php
namespace Aspose\Tasks\WorkingWithProjects;

use com\aspose\tasks\Project as Project;

class VerifyProjectStructure{

    public static function run($dataDir=null)
    {
        # Instantiate project object
        $project = new Project($dataDir . 'test_tasks.mpp');
        if ($project ==true) { //-> check()
            print "Project is ok" . PHP_EOL;
        } else {
            print "Project is not ok" . PHP_EOL;
        }
    }
}
?>